codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionTest.java

```diff
@@ -899,4 +909,22 @@ private void verifyParentChanging(Http2Stream stream, Http2Stream newParent) {
     private void verifyParentChanged(Http2Stream stream, Http2Stream oldParent) {
         verify(clientListener).onPriorityTreeParentChanged(streamEq(stream), streamEq(oldParent));
     }
+    private Http2Stream child(Http2Stream parent, final int id) {
+        try {
+            final Holder<Http2Stream> streamHolder = new Holder<Http2Stream>();
+            parent.forEachChild(new Http2StreamVisitor() {
```

Comment:
Lets promote this to a class which lives in the `Http2StreamVisitor` interface. Something like...

```java
class FindStreamByIdVisitor implements Http2StreamVisitor {
private final int idToFind;
...
}
```